### PR TITLE
Update podified, tempest and tofu zuul jobs to CRC 2.39 - OCP 4.16

### PR DESCRIPTION
**Pull request by lewisdenny**
Jira: https://issues.redhat.com/browse/OSPRH-8665

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running

Jobs to check:
- [x] [podified-multinode-ironic-deployment](https://review.rdoproject.org/zuul/build/fbd61984d8c74ffaa0b123a41b8f59e3)
- [x] cifmw-molecule-tofu - [Result](https://review.rdoproject.org/zuul/build/c4f416942d4a40c2adff0d197436c81b)

### CODE REVIEW

**Comment by lewisdenny** [comment]
recheck

File: zuul.d/tofu.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@
       - ^ci/playbooks/molecule.*
       - ^ci_framework/playbooks/run_tofu.yml
     name: cifmw-molecule-tofu
-    nodeset: centos-9-crc-2-36-0-xl
+    nodeset: centos-9-crc-2-39-0-xl
     parent: cifmw-molecule-base
     vars:
       TEST_RUN: openshift_login
```
Comment (pablintino) [inline zuul.d/tofu.yaml:12]: This thing is wrong, it's testing the openshift_login role :(

Comment (lewisdenny) [reply zuul.d/tofu.yaml:12]: https://github.com/openstack-k8s-operators/ci-framework/pull/2104

Comment (lewisdenny) [reply zuul.d/tofu.yaml:12]: I can see it failing if it hasn't been testing all this time so I created a new PR to fix this.

**Review by raukadah — APPROVED** [review]
/approve

**Pull request merged**